File: src/main/scala/com/deswaef/wowscrappie/WowScrappie.scala
package com.deswaef.wowscrappie

import org.springframework.boot.autoconfigure.{SpringBootApplication, EnableAutoConfiguration}
import org.springframework.context.annotation.{Configuration, ComponentScan}
import org.springframework.data.jpa.repository.config.EnableJpaRepositories
import org.springframework.scheduling.annotation.EnableAsync

/**
 * Created by QuintenDes on 29/07/15.
 */
@EnableJpaRepositories(basePackages = Array("com.deswaef"))
@ComponentScan(basePackages = Array("com.deswaef"))
@EnableAsync
@SpringBootApplication
class WowScrappie {

}

File: src/main/java/com/deswaef/weakauras/WowScrappie.java
package com.deswaef.weakauras;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.actuate.system.ApplicationPidFileWriter;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.context.annotation.ComponentScan;
import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
import org.springframework.scheduling.annotation.EnableAsync;

@EnableJpaRepositories(basePackages = "com.deswaef")
@ComponentScan(basePackages = "com.deswaef")
@EnableAsync
@SpringBootApplication
public class WowScrappie {

    public static void main(String[] args) {
        SpringApplication application = new SpringApplication(WowScrappie.class);
        application.addListeners(new ApplicationPidFileWriter());
        application.run();
    }
}
